worlds/earthbound/src/eb.asm

```diff
@@ -0,0 +1,18861 @@
+db $a5, $a2, $99, $9e, $97, $50, $92, $91, $a4, $a4, $9c, $95, $5e, $03, $00, $08
+db $9b, $6d, $c5, $00, $13, $02;Disable o
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+db $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50, $92, $a2, $9f, $9b
+db $95, $9e, $50, $a0, $9f, $93, $9b, $95, $a4, $a7, $91, $a4, $93, $98, $5e, $03
+db $00, $08, $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03
+db $00, $70, $71, $50, $92, $a2, $9f, $9b, $95, $9e, $50, $a3, $93, $a2, $95, $95
+db $9e, $50, $96, $a2, $9f, $9d, $50, $91, $50, $98, $91, $9e, $94, $98, $95, $9c
+db $94, $50, $94, $95, $a6, $99, $93, $95, $5e, $03, $00, $08, $6f, $6b, $c5, $00
+db $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50, $92, $a2
+db $9f, $9b, $95, $9e, $50, $a2, $91, $aa, $9f, $a2, $5e, $03, $00, $08, $6f, $6b
+db $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50
+db $92, $a2, $9f, $9b, $95, $9e, $50, $94, $95, $a3, $9b, $50, $96, $91, $9e, $5e
+db $03, $00, $08, $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e
+db $03, $00, $70, $71, $50, $92, $a2, $9f, $9b, $95, $9e, $50, $96, $91, $a5, $93
+db $95, $a4, $50, $98, $95, $91, $94, $5e, $03, $00, $08, $6f, $6b, $c5, $00, $13
+db $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50, $a0, $99, $95
+db $93, $95, $50, $92, $a2, $9f, $9b, $95, $9e, $50, $9f, $96, $96, $50, $9f, $96
+db $50, $91, $50, $a4, $a5, $92, $91, $5e, $03, $00, $08, $6f, $6b, $c5, $00, $13
+db $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50, $92, $a2, $9f
+db $9b, $95, $9e, $50, $a3, $a0, $a2, $99, $9e, $9b, $9c, $95, $a2, $5e, $03, $00
+db $08, $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00
+db $70, $83, $9f, $9d, $95, $50, $a0, $91, $a2, $a4, $50, $92, $95, $9c, $9f, $9e
+db $97, $99, $9e, $97, $50, $a4, $9f, $50, $91, $50, $a4, $a2, $9f, $9d, $92, $9f
+db $9e, $95, $5e, $03, $00, $08, $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c
+db $05, $01, $6e, $03, $00, $70, $71, $50, $92, $a2, $9f, $9b, $95, $9e, $50, $9d
+db $99, $93, $a2, $9f, $a7, $91, $a6, $95, $50, $9f, $a6, $95, $9e, $5e, $03, $00
+db $08, $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00
+db $70, $80, $91, $a2, $a4, $50, $9f, $96, $50, $91, $50, $92, $a2, $9f, $9b, $95
+db $9e, $50, $94, $95, $95, $a0, $50, $96, $a2, $a9, $95, $a2, $5e, $03, $00, $08
+db $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70
+db $71, $50, $9e, $9f, $aa, $aa, $9c, $95, $50, $96, $a2, $9f, $9d, $50, $91, $50
+db $a7, $91, $a4, $95, $a2, $99, $9e, $97, $50, $93, $91, $9e, $5e, $03, $00, $08
+db $6f, $6b, $c5, $00, $13, $02, $01, $50, $6c, $1c, $05, $01, $6e, $03, $00, $70
+db $71, $50, $92, $a2, $9f, $9b, $95, $9e, $50, $97, $91, $a2, $94, $95, $9e, $50
+db $98, $9f, $a3, $95, $5e, $03, $00, $08, $6f, $6b, $c5, $00, $13, $02, $01, $50
+db $6c, $1c, $05, $01, $6e, $03, $00, $70, $71, $50, $92, $a2, $9f, $9b, $95, $9e
+db $50, $9
```

Comment:
Won't necessarily ask for a change because it works and runs during a not so important process but for future reference consider using a pointer table if you're just going to keep adding entries to this long if-else chain, becomes a lot more manageable than playing around with branches and their 127 jump limit.